packages/fast-usdc-contract/src/exos/settler.ts

```diff
@@ -251,6 +263,50 @@ export const prepareSettler = (
           assert.typeof(registration, 'object');
           this.state.registration = registration;
         },
+        migrateMintedEarly() {
+          const { self } = this.facets;
+          const { mintedEarly } = this.state;
+          for (const [key, count] of mintedEarly.entries()) {
+            const { address, amount } = parseMintedEarlyKey(key);
+            for (let i = 0; i < count; i += 1) {
+              self.addMintedEarly(address, AmountMath.make(USDC, amount));
+            }
+          }
+          mintedEarly.clear();
```

Comment:
This looks like we read from and write to the same place (`this.state.mintedEarly) in the migration.

Then, after all that work, we clear the store.

Am I reading this right?

Is this code used?

Reply:
Yes, it's migrating how the data in that single place is keyed

Reply:
I still don't understand `mintedEarly.clear()` at the end. Why bother with the loop at all if we're just going to clear the store?